src/ray/gcs/client.cc

```diff
@@ -0,0 +1,45 @@
+#include "ray/gcs/client.h"
+
+#include "ray/gcs/redis_context.h"
+
+namespace ray {
+
+namespace gcs {
+
+AsyncGcsClient::AsyncGcsClient() {}
```

Comment:
Since these are empty, might be cleaner to just remove ctor & dtor declarations and definitions.